### PR TITLE
SHO-79: RegularsInfo Template

### PR DESCRIPTION
**Pull request by saulprl**
## Describe your changes

This PR adds a new component: RegularsInfo.

## Task URL (Asana, Jira, etc...)

[SHO-79](https://cultoperrocafe.nifty.pm/l/4C2fcet8XiwU)

## What was done in this pull request

- [x] Added new component: `<RegularsInfo />`.
- [x] Added new component: `<Numeral />`.
- [x] Refactored `<Subtitle />` to render a `<Numeral />` when the corresponding prop is provided.
- [x] Storybook with `RegularsInfo`'s only variant.
- [x] Added to `frecuentes.tsx`.

## Demo / Screenshot / Video

[SHO-79.webm](https://user-images.githubusercontent.com/39208827/236095462-7e0fcfc7-fce0-4f06-ab28-6a70b14816d5.webm)

#### Regarding the calculation on `regulars-info.component.tsx`

`const renderBlockDivider = isDesktop ? index < arr.length - (2 - (arr.length % 2)) : index < arr.length - 1;`

When on desktop mode, this calculation checks if the current element is **not** the last element (if the array's length is odd) or one of the last two elements (when the array's length is even).

On mobile, it just checks if the current element is not the last element of the array, regardless of its length.

Then, `renderBlockDivider` is used to conditionally render the `<BlockDivider />` at the end of the `<InfoBlock />`.

Here, we only have 3 `InfoBlock`s, and so the `BlockDivider` is only hidden on the third one.
SHO-79-calc-1

In this case (the default one), the last two `InfoBlock`s won't have a `BlockDivider`.
SHO-79-calc-2

### CODE REVIEW

File: app/components/atoms/numeral/numeral.styles.tsx [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+  position: relative;
+  width: 60px;
+`;
+
+export const Label = styled.span`
+  position: absolute;
+
+  color: ${select(({colors}) => colors.background)};
+  font-family: ${select(({fonts}) => fonts.title)};
+  font-size: 44px;
```
Comment (arielplas) [inline app/components/atoms/numeral/numeral.styles.tsx]: 44px seems a little odd.... don't we have something like 36px in our theme? 👀

File: app/components/templates/regulars-info/regulars-info.component.tsx [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+export const RegularsInfo = () => {
+  const isDesktop = useMediaQuery('(min-width: 768px)');
+
+  return (
+    <InfoSection>
+      <Grid>
+        {regularsInfo.map((info, index, arr) => {
+          const renderBlockDivider = isDesktop
+            ? index < arr.length - (2 - (arr.length % 2))
+            : index < arr.length - 1;
```
Comment (arielplas) [inline app/components/templates/regulars-info/regulars-info.component.tsx:25]: What's this calculation for?

Comment (saulprl) [reply app/components/templates/regulars-info/regulars-info.component.tsx:25]: On the original Figma design, on desktop mode, the last two InfoBlocks don't have a divider at the bottom. On handheld mode, it's only the last InfoBlock that doesn't have a divider, so:
- On desktop mode, the calculation checks whether the current element is one of the last two elements in the array if its length is even; otherwise, it checks if it's the last element. **If we ended up with, say, 3 InfoBlocks, only the last one wouldn't render the divider**.
- On mobile, it just checks whether the current element is the last element of the array.

So in summary it really just makes it so we can have a dynamic number of InfoBlocks.

File: app/components/atoms/numeral/numeral.styles.tsx (hunk 1 above)
Comment (saulprl) [reply app/components/atoms/numeral/numeral.styles.tsx]: Yeah, I forgot to update that hardcoded value, thanks!

File: app/components/templates/regulars-info/regulars-info.component.tsx (hunk 2 above)
Comment (arielplas) [reply app/components/templates/regulars-info/regulars-info.component.tsx:25]: Im going to trust you on this one, because i don't know. 😄

**Review by imariel2d — APPROVED** [review]
